### PR TITLE
Update what-if-my-subscription-expires.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the Note to reflect NCE and Legacy expiration date

fixes#https://github.com/MicrosoftDocs/microsoft-365-docs-pr/issues/23777

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/commerce/subscriptions/what-if-my-subscription-expires.md [hunk 1]
```diff
@@ -125,7 +125,7 @@ At this point, you can't reactivate the subscription. However, as a global or bi
 > [!NOTE]
 >
 > - Adding a new subscription of the same type that was deleted doesn't restore the data that was associated with the deleted subscription.
-> - If a Cloud Solution Provider (CSP) license is suspended, there's no 30-day Expired stage, and services are disabled immediately. Data is deleted after 90 days if the tenant isn't reactivated by adding a new license.
+> - If a legacy Cloud Solution Provider (CSP) license is suspended, there's no 30-day Expired stage, and services are disabled immediately. Data is deleted after 90 days if the tenant isn't reactivated by adding a new license. For New Commerce (NCE) licenses sold via CSPs the same "expired" state as described in this article applies when subscription are not renewed. See [Subscription Lifecycle](/partner-center/subscription-lifecycle) for more information. When NCE licenses in CSP are suspended, they licenses move into disabled state until 90 days after original subscription term end date.
```
Comment (JohanFreelancer9) [inline microsoft-365/commerce/subscriptions/what-if-my-subscription-expires.md]:
```suggestion
> - If a legacy Cloud Solution Provider (CSP) license is suspended, there's no 30-day Expired stage, and services are disabled immediately. Data is deleted after 90 days if the tenant isn't reactivated by adding a new license. For New Commerce (NCE) licenses sold via CSPs, the same "expired" state as described in this article applies when subscriptions are not renewed. See [Subscription Lifecycle](/partner-center/subscription-lifecycle) for more information. When NCE licenses in CSPs are suspended, their licenses move into a disabled state until 90 days after the original subscription term end date.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@cmcatee-MSFT Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@cmcatee-MSFT Gentle reminder.

**Comment by cmcatee-MSFT** [comment]
Other updates to the article addressed this.

**Pull request closed without merging**